### ISSUE TITLE
Add weekly game description of "Epic Games" freebies

### ISSUE THREAD

**Issue by therealmistericraft** [therealmistericraft/MisteriBot#2]
Try to get the game description of all new free games by Epic Games from the HTML-Code and send a Message with it in the Channel:

> Name: #epic-games-news
> ID: 720674929540071564

_**Problem:**_
This would be a server specific feature. Maybe I need a _"setup"-feature_

**Comment by joseywoermann** [comment]
You would need the EpicGames-API in order to do that. Discord-bots can't read content from a website without a specific API.

**Comment by therealmistericraft** [comment]
That has nothing to do with Discord. It's just python code.

**Comment by joseywoermann** [comment]
You'd probably still need an API.

**Comment by therealmistericraft** [comment]
I do not think so, there is a python package to download html files and finding the correct text is `string.split()` and `string.replace()`,
